sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -397,14 +398,14 @@ def list_entities(
             self,
             **kwargs  # type: Any
     ):
-        # type: (...) -> ItemPaged[TableEntity]
+        # type: (...) -> ItemPaged[azure.data.tables.TableEntity]
         """Lists entities in a table.
 
         :keyword int results_per_page: Number of entities per page in return ItemPaged
-        :keyword Union[str, list(str)] select: Specify desired properties of an entity to return certain entities
+        :keyword Union[str,list(str)] select: Specify desired properties of an entity to return certain entities
         :return: Query of table entities
-        :rtype: ItemPaged[TableEntity]
-        :raises: ~azure.core.exceptions.HttpResponseError
+        :rtype: ItemPaged[azure.data.tables.TableEntity]
```

Comment:
In docstrings, I think the types need the `~` for linking: `ItemPaged[~azure.data.tables.TableEntity]`

Reply:
@seankane-msft can you also check if `ItemPaged` needs to be linked with the full path, i.e. if the doc actually needs to be

`~azure.core.paging.ItemPaged[~azure.data.tables.TableEntity]`

Reply:
@iscai-msft yes you are correct. I will update these